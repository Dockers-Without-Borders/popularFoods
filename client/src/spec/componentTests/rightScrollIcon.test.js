import React from 'react';
import { shallow } from 'enzyme';
import RightScrollIcon from '../../components/carousel/icons/rightScrollIcon.jsx';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

describe('<RightScrollIcon/>', () => {
    it('should render', () => {
        const component = shallow(<RightScrollIcon />);
        expect(component.exists()).toEqual(true);
    });
    it('should render properly - snapshot version', () => {
        const component = renderer.create(<RightScrollIcon />);
        expect(component).toMatchSnapshot();
    });
})