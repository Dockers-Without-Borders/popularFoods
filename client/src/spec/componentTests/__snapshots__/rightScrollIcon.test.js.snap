// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RightScrollIcon/> should render properly - snapshot version 1`] = `
<svg
  className="prefix__icon_svg"
  height={24}
  width={24}
>
  <path
    d="M9.525 5.636L8.11 7.05 13.06 12l-4.95 4.95 1.415 1.414L15.89 12 9.524 5.636z"
  />
</svg>
`;
